src/components/Rank/InvalidRankCardItem.tsx

```diff
@@ -44,7 +44,7 @@ export const InvalidRankCardItem = ({
           >
             <Text
               color="inactiveText"
-              fontSize={{ md: 'xs', base: 'sm' }}
+              fontSize={{ base: 'xs' }}
```

Comment:
```suggestion
              fontSize="xs"
```
I think this is same as 'xs'